src/lib.rs

```diff
@@ -79,6 +79,12 @@ pub use crate::crand::ComplexDistribution;
 /// ```
 #[derive(PartialEq, Eq, Copy, Clone, Hash, Debug, Default)]
 #[repr(C)]
+#[cfg_attr(
+    feature = "rkyv",
+    derive(rkyv::Archive, rkyv::Serialize, rkyv::Deserialize,),
+    archive(as = "Complex<T::Archived>", bound(archive = "T: rkyv::Archive"))
```

Comment:
I think this `bound` is redundant with the automatic bounds, from what I see in `cargo expand`.